src/OSPSuite.R/Services/ContainerTask.cs

```diff
@@ -169,30 +179,28 @@ public IParameter[] AllParametersForSensitivityAnalysisMatching(ISimulation simu
 
       public void SetValueByPath(IModelCoreSimulation simulation, string path, double value, bool throwIfNotFound)
       {
-         if (path.Contains(WILD_CARD))
-            throw new OSPSuiteException(Error.CannotSetValueByPathUsingWildCard(path));
-
-         var pathArray = path.ToPathArray();
-         var quantity = simulation.Model.Root.EntityAt<IQuantity>(pathArray);
-         if (quantity != null)
-         {
-            quantity.Value = value;
+         var quantity = singleQuantityByPath(simulation, path, throwIfNotFound);
+         if (quantity == null)
             return;
-         }
 
-         if (throwIfNotFound)
-            throw new OSPSuiteException(Error.CouldNotFindQuantityWithPath(path));
+         quantity.Value = value;
+      }
 
-         _logger.AddWarning(Error.CouldNotFindQuantityWithPath(path));
+      public double GetValueByPath(IModelCoreSimulation simulation, string path, bool throwIfNotFound)
+      {
+         var quantity = singleQuantityByPath(simulation, path, throwIfNotFound);
+         if (quantity == null)
```

Comment:
there is the handy new (ish) syntax
return `quantity?.Value ?? double.NaN`

Don't have to be used but I wanted to point this out just in case